hack/test-end-to-end-docker.sh

```diff
@@ -15,9 +15,8 @@ echo "[INFO] Starting containerized end-to-end test"
 
 unset KUBECONFIG
 
-TMPDIR="${TMPDIR:-"/tmp"}"
-BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-e2e-containerized}"
 setup_env_vars
+setup_temp_dir_vars "test/end-to-end/containerized/"
```

Comment:
`BASETMPDIR=/tmp/openshift/origin`
`TMPDIR=${BASETMPDIR}/${that-arg}`
so the result of this is
`TMPDIR=/tmp/openshift/origin/test/end-to-end/containerized`